test/stripe/api_resource_test.rb

```diff
@@ -414,6 +463,19 @@ class NestedTestAPIResource < APIResource
         acct.save
       end
 
+      should "update nothing if nothing changes" do
+        Stripe::Account.construct_from(id: "acct_id",
+                                       metadata: {
+                                         key: "value",
+                                       })
+
+        stub_request(:post, "#{Stripe.api_base}/v1/accounts/acct_id")
+          .with(body: {})
+          .to_return(body: JSON.generate("id" => "acct_id"))
+
+        Stripe::Account.update("acct_id")
+      end
+
```

Comment:
Don't think this test case make sense as it relates to `.save`'s diffing behavior.
```suggestion
      should "update nothing if nothing changes" do
        Stripe::Account.construct_from(id: "acct_id",
                                       metadata: {
                                         key: "value",
                                       })

        stub_request(:post, "#{Stripe.api_base}/v1/accounts/acct_id")
          .with(body: {})
          .to_return(body: JSON.generate("id" => "acct_id"))

        Stripe::Account.update("acct_id")
      end

```